plugins/main/bridge/bridge.go

```diff
@@ -240,7 +240,7 @@ func calcGateways(result *current.Result, n *NetConf) (*gwInfo, *gwInfo, error)
 
 		// Add a default route for this family using the current
 		// gateway address if necessary.
-		if n.IsDefaultGW && !gws.defaultRouteFound {
```

Comment:
Are we sure this is NOP? At this point `gws` refers to a global defined at the top of the function:

```
	gwsV4 := &gwInfo{}
	gwsV6 := &gwInfo{}
```

So if there are multiple v4/v6 addresses, I think defaultRouteFound would carry across the IP addressing loop. I think we need to keep this?